uiowa/uiowa
# updating allowed blocks for #1889

This is for #1889 and it is to limit the number of possible blocks a user is presented with when building a page. I do not think this list is definitive, but at least it is a start to what we could work towards. This does not account for [this](https://github.com/uiowa/uiowa/issues/1789#issuecomment-653075625) as my reasoning is that if paragraphs goes away, those might go away and then we might have to rethink our strategy? Either way, this should start the ball rolling.

To test:
1. blt dsa a layout builder site, composer install
2. check that you can only add a limited number of blocks
3. additionally, you can check at this location `/admin/structure/types/manage/page/display#edit-layout-layout-builder-restrictions-allowed-blocks` to see if there is configuration being set for every option.
